ctapipe/reco/tests/test_HillasReconstructor.py

```diff
@@ -82,7 +82,8 @@ def test_h_max_results():
     # np.testing.assert_allclose(fitted_core_position.value, [0, 0], atol=1e-3)
 
 
-def test_reconstruction():
+@pytest.mark.parametrize("max_events, reco_num_events_expected", [(10, 0)])
```

Comment:
This two paremeters can be fixed, right? I thought about using something like this:
```
@pytest.mark.parametrize("reconstructor", [HillasIntersection, HillasReconstructor])
```

Reply:
I wrote a new test which run both reconstructions.
I used a feature called pytest.fixture to collect the functions and to pass them to the test. If I ask to have at least 40 telescopes in the `hillas_dict`, the test fails...which means it works properly.